tools/importer-rest-api-specs/components/parser/models.go

```diff
@@ -638,7 +638,7 @@ func (d SwaggerDefinition) parseNativeType(input *spec.Schema) *models.ObjectDef
 		}
 	}
 
-	if input.Type.Contains("file") {
+	if input.Type.Contains("file") || (input.Format == "file" && input.Type.Contains("object")) {
```

Comment:
we need this to be case-insensitive due to inconsistencies in the Swagger

```suggestion
	if input.Type.Contains("file") || (strings.EqualsFold(input.Format, "file") && input.Type.Contains("object")) {
```